.circleci/config.yml

```diff
@@ -234,7 +268,7 @@ workflows:
       - visual-test:
           node-version: lts/gallium
       - e2e-couchdb:
-          node-version: lts/gallium
+          node-version: lts/hydrogen
```

Comment:
cache conflict during testing

Reply:
Update: Just moved them all to 18 so we could get npm dep resolution speed improvements